setuptools_rust/build.py

```diff
@@ -84,6 +86,15 @@ def build_extension(self, ext):
             }
         )
 
+        # If we are on a 64-bit machine, but running a 32-bit Python, then
+        # we'll target a 32-bit Rust build.
+        # TODO: make this not Windows specific
```

Comment:
Might be tempted to describe the motivation of the TODO.

```suggestion
        # TODO: include --target for all platforms so env vars can't break the build
```